pom.xml

```diff
@@ -13,8 +13,8 @@
         <url>https://jira.csiro.au/browse/AUS</url>
     </issueManagement>
     <ciManagement>
-        <system>Jenkins</system>
-        <url>https://cgsrv8.arrc.csiro.au/jenkins</url>
+        <system></system>
```

Comment:
Should this whole <ciManagement> tag be removed?

Reply:
Fixed.